gui/themes/scummtheme.py

```diff
@@ -85,7 +85,7 @@ def parseSTX(theme_file, def_file, subcount):
 	return subcount
 
 def buildDefTheme(themeName):
-	def_file = open("default.inc", "w")
+	def_file = open("default.inc", mode="w", newline="\n")
```

Comment:
Without that, when writing in text mode, newlines default to the system newline format, which is `\r\n` on Windows.  This means that generating default.inc on Windows will change the EOL format for the entire file, which causes diff tools to treat it as a complete file replacement instead of showing just the actually-changed lines, which in turn was causing the problem of making it MUCH harder to read the commit diff and figure out if it was generated from the correct theme file.